consultation_analyser/consultations/api/views/consultation.py

```diff
@@ -55,3 +60,36 @@ def demographic_options(self, request, pk=None):
         serializer = DemographicOptionSerializer(instance=data, many=True)
 
         return Response(serializer.data)
+
+
+@api_view(["GET"])
+def get_consultation_folders(request) -> HttpResponse:
+    """
+    get consultation folders.
+    """
+    consultation_folders = ingest.get_consultation_codes()
+
+    return JsonResponse(consultation_folders, safe=False)
+
+
+@api_view(["POST"])
+def submit_consultation_import(request) -> HttpResponse:
+    """
+    Submit consultation import.
+    """
+    consultation_name = request.data.get("consultation_name")
+    consultation_code = request.data.get("consultation_code")
+    timestamp = request.data.get("timestamp")
+    action = request.data.get("action")
+
+    try:
+        import_consultation_job.delay(
+            consultation_name=consultation_name,
+            consultation_code=consultation_code,
+            timestamp=timestamp,
+            current_user_id=request.user.id,
+            sign_off=action == "sign_off",
+        )
+        return HttpResponse(status=201)
+    except Exception as e:
+        return HttpResponse(status=500, content=str(e))
```

Comment:
lets not expose the stack trace
```suggestion
        return HttpResponse(status=500, content="something went wrong")
```